Cardoon/src/components/pages/CardFormPage/CardFormPage.tsx

```
@@ -191,36 +225,42 @@ export default () => {
+          <div className="CardFormPage__modal-questions">
+            {isGenerationLoading && <Loader />}
+            {subQuestions &&
+              subQuestions.map(({ question, answer }, index) => (
+                <QuestionLine
+                  key={index + question + answer}
+                  question={question}
+                  answer={answer}
+                  category={newCard.category}
+                />
+              ))}
+          </div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)